### PR TITLE
52697-user-take-test

### PR DESCRIPTION
**Pull request by l0ngvh**
## Related Tickets
- [#52697](https://edu-redmine.sun-asterisk.vn/issues/52697)

## WHAT (optional)
- I allow user to take test created for each lesson.

## HOW
- By using the edit routes of exams controller as the page the exams take place.

## WHY (optional)
- Because a created test shouldn't be modified to prevent cheating, the exams edit routes is vacant, so it is repurpose to be the exam taking roues

## Evidence (Screenshot or Video)
image
image

### CODE REVIEW

**Review by l0ngvh — COMMENTED** [review]
ready

File: app/views/exams/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+          <div class="card-header font-family-ubuntu">
+            <%= question_fields.label :definition, question.true_answer.definition %>
+          </div>
+          <div class="card-body">
+            <div class="exam-question-body">
+              <% answers = [question.true_answer] + question.wrong_answers %>
+              <% answers.shuffle().each do |answer| %>
+                <div class="d-flex justify-content-center align-items-center">
+                  <%= question_fields.radio_button :user_answer_id, answer.id, class: "btn-check" %>
+                  <%= question_fields.label :user_answer_id, answer.translation, value: answer.id, class: "btn btn-outline-secondary w-75 h-75 p-4 d-flex align-items-center justify-content-center" %>
```
Comment (htpt2910) [inline app/views/exams/edit.html.erb]: xuống dòng đeee

**Review by htpt2910 — COMMENTED** [review]
cmt

**Comment by l0ngvh** [comment]
ready

**Pull request merged**